main/zoe/api/zoe.md

```diff
@@ -0,0 +1,77 @@
+# Zoe
+
+Zoe is a long-lived and well-trusted contract that enforces offer safety for the contracts that use it. Zoe has a single `seatAssay` and a single `escrowReceiptAssay` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `seatAssay` or `escrowReceiptAssay` and thus validate any `seat` or `escrowReceipt` they receive from someone else.
+
+### <span style="color:red">Zoe and ZoeGoverningContractFacet have some similar methods so those examples are the same. Those should be double-checked to make sure they are accurate in both locations. Methods: getSeatAssay, getEscrowReceiptAssay.</span>
+
+
+## zoe.getSeatAssay()
+- Returns: `{Assay}`
+
+Get the long-lived `seatAssay`. The mint associated with the `seatAssay` creates the ERTP payments that represent the right to claim the payouts of involvement in a contract.
+
+### <span style="color:red">Couldn't find an example in the code, this needs checking.</span>
+```js
+const someSeatAssay = zoe.getSeatAssay()
+```
+
+## zoe.getEscrowReceiptAssay()
+- Returns: `{Assay}`
+
+Get the long-lived `escrowReceiptAssay`. The mint associated with the `escrowReceiptAssay` creates the ERTP payments that represent proof of escrow and may be passed into a smart contract method.
+
+```js
+const escrowReceiptAssay = zoe.getEscrowReceiptAssay();
+```
+
+## zoe.getAssaysForInstance(installationHandle)
+- `installationHandle` `{Object}`
+- Returns: `{Array <Assay>}`
+
+Returns the array of assays for the particular instance. This is helpful in the case that the user has forgotten the order.
+
+```js
+const contractAssays = await E(zoe).getAssaysForInstance(instanceHandle);
+```
+
```

Comment:
I had missed this, but let's add zoe.install

```
  /**
   * Create an installation by safely evaluating the code and
   * registering it with Zoe. Returns an installationHandle.
   */
  install(code :String) -> Object
```

Reply:
Added. This is the example I used:

```
import bundleSource from '@agoric/bundle-source';

// Pack the contract.
const { sourceCode, moduleFormat } = await bundleSource(someContract);

// create an `installationHandle` for someContract
const installationHandle = zoe.install(sourceCode, moduleFormat);
```